Tests/APITesters/CustomEntitlementComputationSwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -101,7 +101,7 @@ private func checkTypealiases(
                                                   userCancelled: userCancelled)
 
     // swiftlint:disable:next line_length
-    let purchaseCompletedBlock: PurchaseCompletedBlock = { (_: StoreTransaction?, _: CustomerInfo?, _: Error?, _: Bool) -> Void in }
+    let purchaseCompletedBlock: PurchaseCompletedBlock = { (_: StoreTransaction?, _: CustomerInfo?, _: Error?, _: Bool) in }
```

Comment:
I kinda disagree with swiftlint on this one, the `-> Void` makes it much clearer that you're looking at a block with void return type IMO, at first glance this seems like a tuple otherwise